scss/os/_overrides_tuieditor.scss

```diff
@@ -499,10 +602,41 @@
   }
 }
 
+.te-md-container {
+  .CodeMirror {
+    color: inherit;
+    font-family: $font-family-monospace;
+  }
+}
+
 .te-ww-mode .te-ww-container {
   z-index: 0;
 }
 
 .tui-tooltip {
   z-index: $zindex-tooltip;
 }
+
+.tui-md-code {
+  background: none;
+}
+
+.tui-md-code.tui-md-marked-text {
+  color: #f89406;
```

Comment:
prefer variables over strings for colors

Reply:
Color codes have been replaced with variables.